ogre2/include/ignition/rendering/ogre2/Ogre2DepthCamera.hh

```diff
@@ -119,6 +119,9 @@ namespace ignition
       /// ogre camera has not been created.
       public: double FarClipPlane() const override;
 
+      // Documentation inherited.
+      public: void AddRenderPass(const RenderPassPtr &_pass) override;
```

Comment:
The ABI checker is complaining about this. Was it ok to add it to `ign-rendering3`?

Reply:
I don't remember if the issue was there when merging rendering2 to rendering3 but it could have slipped through. I started trying to workaround this but realized that this would be removing a public function that was previously added in rendering2 and rendering3.  The ABI report shows that the difference is in 2 entries in the vtable related to `AddRenderPass()`, which is that now the derived `Ogre2DepthCamera::AddRenderPass` function will be called instead of the base class. It's the intended behavior change but I don't know if that'll cause undefined behavior for downstream libraries. We can either

1. Leave the function here as it is with the ABI issue

1. Remove the function and find a workaround in order keep the depth noise feature (that relies on the AddRenderPass) and update the migration guide to explain reason behind the function removal. 

I'm tempted to go with 1. but either way works.

Reply:
I left a comment on the offending PR here: https://github.com/ignitionrobotics/ign-rendering/pull/122#issuecomment-717560255

At this point, I guess we may as well go with option 1. I think we should add a note to the migration guide about the breakage for all versions from Blueprint to Citadel.